pkg/authorization/authorizer/authorizer_test.go

```diff
@@ -440,7 +440,7 @@ func newDefaultGlobalBinding() []authorizationapi.PolicyBinding {
 			Namespace: bootstrappolicy.DefaultMasterAuthorizationNamespace,
 		},
 		RoleBindings: map[string]authorizationapi.RoleBinding{
-			"cluster-admins": {
+			"extra-cluster-admins": {
```

Comment:
The test adds an additional rolebinding to make sure that merging works correctly.  The names must be unique between that and the bootstrap name.  I updated the bootstrap policy role binding names to make more sense:  `cluster-admin` role gets bound under `cluster-admins`